IntercomDotnet/Resources/Users.cs

```diff
@@ -47,7 +47,7 @@ public dynamic DeleteWithUserId(string userId)
         [Obsolete("userId is a string in the Intercom API so please use the string-based overload going forward.")]
         public dynamic Get(string email = null, int? userId = null)
         {
-            return Get(email, userId?.ToString());
+            return Get(email, userId.ToString());
```

Comment:
I am using Visual studio 2013, .net framework : 4.5 version. as we are already using IsNullOrEmpty() check I fixed this with Convert.ToString().

Let me know if I can contribute more API's in intercom :)